### ISSUE TITLE
Load Image Docs Link Should Not Be Displayed (as it doesn't exist)

### ISSUE THREAD

**Issue by HarshKhandeparkar** [publiclab/image-sequencer#1525]
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open-source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### What is to be done?
The first step in every sequence of ImageSequencer is the `load-image` step where the input image is loaded. This is just a sequencer method and not an actual module. It has no dedicated documentation.

All the other modules of ImageSequencer have their own docs [here](https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md)

Even though `load-image` has no docs, there is a link in the UI for `load-image` which leads to nothing.
![Screenshot from 2020-01-17 23-35-45](https://user-images.githubusercontent.com/34770591/72636459-1d1b8d80-3985-11ea-9162-c40fe2456cd4.png)

Files to refer to:
1. https://github.com/publiclab/image-sequencer/blob/d1b9b8beccdbe5ba98cf4e352af103b4913a5fe6/examples/lib/defaultHtmlStepUi.js#L48-L50

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 Modify all the files where this code is found.

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need it if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

**Comment by pcube99** [comment]
I want to work on this issue.
can you elaborate, what I am supposed to do in this ?

**Comment by HarshKhandeparkar** [comment]
You have to make it so that only the first step has no link.

**Comment by HarshKhandeparkar** [comment]
All the other steps' description should be clickable.

**Comment by pcube99** [comment]
so I just need to put an If statement or an ternary operator and check that step.name is load-image or not.

If it is load-image then remove the link otherwise redirect it.

**Comment by HarshKhandeparkar** [comment]
Yeah. Great! You can ask for help here. We are always ready to help.

**Comment by pcube99** [comment]
@HarshKhandeparkar  please check the PR.

**Comment by HarshKhandeparkar** [comment]
This one is open :)

**Comment by ashirbad29** [comment]
I want to work on this issue, how should i proceed?

**Comment by HarshKhandeparkar** [comment]
Just follow the steps mentioned in the issue body @ashirbad29 :) If there is any doubt, please ask.

**Comment by HarshKhandeparkar** [comment]
In short, as you can see in that code, it tries to *generate* a link to the docs but the link doesn't always exist. Instead of dynamically generating, you can use `step['docs-link']` because all the links have been manually written under that parameter.
Just check if `step['docs-link']` exists, if it does then only show the link.

**Comment by ashirbad29** [comment]
since its a small issue i change the code in the browser, should I need to test the code first by npm-run-test

**Comment by ashirbad29** [comment]
@HarshKhandeparkar please review my PR :)

**Comment by HarshKhandeparkar** [comment]
> since its a small issue i change the code in the browser, should I need to test the code first by npm-run-test

Not really, I don't think that part even has a test.

**Comment by ashirbad29** [comment]
@HarshKhandeparkar I worked on your suggestion and made changes, please review it

**Comment by GuptaTanisha** [comment]
@HarshKhandeparkar Can I work on this?

**Comment by HarshKhandeparkar** [comment]
It was actually fixed by the other PR but the PR has a conflict. Maybe you can revive that PR?

**Comment by SharmaTanish** [comment]
Hi @HarshKhandeparkar  ! I am very exited to work on this issue , if it is still open . I have learnt web development from coding ninjas and all my basic concepts are clear . I am very hardworking and have scored 94.2% in 12th . You can also go through my linkedIn profile - https://www.linkedin.com/in/tanish-a54b771b3 .
Looking forward for your reply .

**Comment by HarshKhandeparkar** [comment]
Sure!

**Comment by pawanmkr** [comment]
is someone working on this issue? or having problems solving it? i also want to solve this issue

**Comment by Demon-Sheriff** [comment]
@harshkhandeparkar  if the issue is still open, I would like to work on this

**Comment by faratabassum09-a11y** [comment]
Hi! I'm Faraha, a CS student contributing to Public Lab. 
I'd like to work on this issue. Can someone guide me 
to the relevant file? Thank you!

**Comment by faratabassum09-a11y** [comment]
Hi! I've submitted a PR for this issue.
PR link: #2211
Please review when you get a chance. Thank you!
